contributing-guides/style-guide.md

```diff
@@ -182,15 +182,12 @@ In this case, provide a note and method to determine whether the command current
 
 ## Option syntax
 
-- Use **GNU-style long options** (like `--help` rather than `-h`) when they are cross-platform compatible (intended to work the same across multiple platforms).
+- For commonly/frequently used commands (e.g. `grep`, `tar`, `etc`), we prefer using short options along with [mnemonics](#short-option-mnemonics).
```

Comment:
This sounds like the `{{-short|--long}}` syntax is not allowed on frequently used pages. Do we want to disallow it?

Reply:
Nope, I will reword this to use wording instead.